### ISSUE TITLE
Auswahl über Kunden - bei mehr als 20 Projekten 

### ISSUE THREAD

**Issue by Christian-H** [yakamara/redaxo_prozer#215]
Bei mehr als 20 Projekten wird bei der Projektauswahl (rechts oben) nach Kunden die Auswahl angeboten. Dies hätten wir gerne auch für die weiteren Projektauswahl - Buttons, wie bei Terminen/Jobs buchen, Suche Projektauswahl, Ablage E-Mails in Projekten, ...

**Comment by tbaddade** [comment]
Mir geht es eher genau anders herum. Ich hätte lieber die komplette Projektliste oben rechts und nicht die Auswahl nach Kunden (alphabetische Sortierung fehlt).

**Comment by Christian-H** [comment]
Ja, die alpahbetische Sortierung fehlt. Soll ich dafür ein Issue aufmachen? 
Thomas, auf wieviel projekte bist du zugelassen? Ich bin auf 110 Projekten zugelassen und von daher sehr froh über die Kundenliste. 
Bei Zeitbuchungen brauche ich das nicht, da mit der Tastenauswahl, wie "Pro" dann schnell das "Projekt" ausgewählt wird.

**Comment by tbaddade** [comment]
> Thomas, auf wieviel projekte bist du zugelassen? Ich bin auf 110 Projekten zugelassen und von daher sehr froh über die Kundenliste. 

bin grad bei 31. Und da finde ich es noch nicht sinnvoll.
Ich hatte schon mal Ideen zu diesem Verhalten notiert. siehe #130
